roles/kubernetes/master/templates/kubeadm-config.v1beta1.yaml.j2

```diff
@@ -0,0 +1,258 @@
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: InitConfiguration
```

Comment:
Cf https://github.com/kubernetes-sigs/kubespray/pull/3814 should be added in InitConfiguration